### DISCUSSION TITLE
Diablo 2 (devilutionX 2)

### DISCUSSION THREAD

**Discussion by micriv** [diasurgical/DevilutionX#1627] [Q&A] [answered]
Thank you very much for the work you have done with Diablo. I spend a lot of hours right now replaying my favorite game since I was a kid.

The sad news is that one day, I will completely finish the game and at the same time, I was wondering if you didn't intend to do the same for Diablo 2?

Thank you very much and have a good day! :)

**Comment by AJenbo** [comment, marked as answer]
Glad you enjoy it. We have no interest in work on Diablo 2.

**Reply by micriv** [reply]
I understand. Thank you for your answer! 😉

**Comment by julealgon** [comment]
There are attempts of reverse engineering D2. The most advanced one i know of is MetalStorm. You might want to check that out.

**Reply by AJenbo** [reply]
Is that still publicly available?

**Reply by julealgon** [reply]
It was never public. I think the last updates from Nefarius are several years old.

**Reply by AJenbo** [reply]
yeah the latest I could find was from 2016, but apparently, as late as last year he is still working on it.

**Reply by M0Rf30** [reply]
Nowadays [OpenDiablo2](https://github.com/OpenDiablo2/OpenDiablo2) is a great attempt to provide an open Diablo 2 implementation (they will move to [AbyssEngine](https://github.com/OpenDiablo2/AbyssEngine))
